docs/abi-compatibility.md

```diff
@@ -0,0 +1,35 @@
+# Windows
```

Comment:
Besides a windows section, we should have some kind of call out that compiling with the STDLiB or ABSEIL has no binary compatibility guarantees.